Graft Network Windows Compile.md

```diff
@@ -0,0 +1,148 @@
+
+# GRAFT COMMUNITY
+### Windows Compile Instructions For GRAFT NETWORK
+##### By Patrick Durbin @SomethingGettingWrong
+![GClogo](https://github.com/Patrickdurbin/docs/blob/Windows-Compile/windows-compile-images/GClogo.jpg)
+
+First head on over to the msys64 website.[MSYS2.ORG](https://www.msys2.org/)
```

Comment:
This looks a bit odd without a space between the . and the MSYS2.ORG link

Reply:
I can commit to adding a space I suppose